Assets/SEE/Layout/NodeLayouts/IncrementalTreeMap/Rectangle.cs

```diff
@@ -0,0 +1,70 @@
+namespace SEE.Layout.NodeLayouts.IncrementalTreeMap
+{
+    /// <summary>
+    /// A class that represents a rectangle shape as part of a layout
+    /// </summary>
+    internal class Rectangle
```

Comment:
The value type behaviour of a struct would probably change a lot of code. 
I gave up the idea of using a struct in the early development phase because the reference type behaviour of a class fits the usage better, imo.
So if you don't see a major conflict here, i wouldnt change it.

Reply:
Sure, if the value type semantics break existing code, we can keep it as is.